qiita_db/metadata_template/prep_template.py

```diff
@@ -111,7 +108,7 @@ def create(cls, md_template, raw_data, study, data_type,
 
         # Get a connection handler
         conn_handler = SQLConnectionHandler()
-        queue_name = "CREATE_PREP_TEMPLATE_%d" % raw_data.id
+        queue_name = "CREATE_PREP_TEMPLATE_%d" % study.id
```

Comment:
Good point but this will not be the case if we allow for multi-threaded tornado or if we move this section to run as a separate IPython job. Better safe than sorry, no?